src/llamafactory/train/sft/workflow.py

```diff
@@ -33,6 +33,9 @@ def run_sft(
     dataset = get_dataset(model_args, data_args, training_args, stage="sft", **tokenizer_module)
     model = load_model(tokenizer, model_args, finetuning_args, training_args.do_train)
 
+    if data_args.efficient_packing:
+        configure_packing(model.config, model_args)
```

Comment:
could we do `configure_packing` in `llamafactory.model.patcher`?

Reply:
Sure, I just edited it